distribution/docs/src/assembly/jdocs.xml

```diff
@@ -32,5 +32,14 @@
             </includes>
             <outputDirectory></outputDirectory>
         </fileSet>
+        <fileSet>
+             <fileMode>0644</fileMode>
+             <directoryMode>0755</directoryMode>
```

Comment:
❓ These values seem to be the default (User read-write, Group and Other read-only). Do you need to specify them?

Reply:
I'll find out